### PR TITLE
use the clock mock for progress indicator tests

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Branch? | 3.0 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | https://github.com/symfony/symfony/issues/15617#issuecomment-196722189 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: phpunit [hunk 1]
```diff
@@ -11,7 +11,7 @@
  */
 
 // Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2016-03-16 15:36 UTC
+// Cache-Id-Version: 2016-03-16 16:33 UTC
```
Comment (nicolas-grekas) [inline phpunit]: no need to change, the cached one is good

File: phpunit.xml.dist [hunk 2]
```diff
@@ -52,6 +52,7 @@
         <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener">
             <arguments>
                 <array>
+                    <element key="time-sensitive"><string>Symfony\Component\Console</string></element>
```
Comment (nicolas-grekas) [inline phpunit.xml.dist]: Needs to be changed like in https://github.com/symfony/symfony/blob/master/phpunit.xml.dist#L59

**Comment by xabbuh** [comment]
failure on AppVeyor is unrelated

File: phpunit (hunk 1 above)
Comment (xabbuh) [inline phpunit]: I thought this is necessary every time we change the PHPUnit configuration file, isn't it?

File: phpunit.xml.dist (hunk 2 above)
Comment (xabbuh) [inline phpunit.xml.dist]: The component's config file can be kept as is or do I need to update that too?

File: phpunit (hunk 1 above)
Comment (nicolas-grekas) [inline phpunit]: it's required when you need to force a reload of phpunit or any of its dependencies, not the case here

File: phpunit.xml.dist (hunk 2 above)
Comment (nicolas-grekas) [inline phpunit.xml.dist]: the component having only one namespace is fine

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request merged**